src/OrchardCore/OrchardCore.Abstractions/Json/Serialization/DateTimeJsonConverter.cs

```diff
@@ -18,7 +18,13 @@ public override DateTime Read(ref Utf8JsonReader reader, Type typeToConvert, Jso
 
         if (!reader.TryGetDateTime(out var value) && DateTime.TryParse(reader.GetString()!, out value))
         {
-            return value;
+            var stringValue = reader.GetString();
+            if (DateTime.TryParse(stringValue, out value))
```

Comment:
This looks buggy, aren't you reading the next token here? The value is already parsed at line 19.

Reply:
Fixed